### ISSUE TITLE
Move HTML documentation under /docs/

### ISSUE THREAD

**Issue by cvrebert** [w3c/css-validator#33]
Currently it clutters up the root directly something fierce.

**Comment by xfq** [comment]
Agreed. BTW, [BUGS.html](https://github.com/w3c/css-validator/blob/master/BUGS.html) and [TODO.html](https://github.com/w3c/css-validator/blob/master/TODO.html) are almost identical.
